shlr/bochs/include/libbochs.h

```diff
@@ -28,7 +35,6 @@ typedef struct libbochs_t {
 //DWORD WINAPI MyThLector_(LPVOID lpParam)
 //DWORD WINAPI MyThEscritor_(LPVOID lpParam)
 bool WaitForReply_(libbochs_t *b);
-int RunRemoteThread_(libbochs_t* b, const ut8* lpBuffer, ut32 dwSize, int a4, ut32 *lpExitCode);
 void ResetBuffer_(libbochs_t* b);
 bool CommandStop_(libbochs_t * b);
```

Comment:
all those methods must be bochs_* but if they are not used externally must be killed